### PR TITLE
Updated *.nix files

### PR DESCRIPTION
**Pull request by alexprivalov**
###### Description of changes

 - updated dependencies, forced enabled network manager 

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/networking/twingate/default.nix
Comment (tonyshkurenko) [inline pkgs/applications/networking/twingate/default.nix:16]: For other reviewers: we will change version here and URL below when we release linux client to master (to put prod values here)

**Review by tonyshkurenko — APPROVED** [review]

**Comment by alexprivalov** [comment]
@sagie-shamay 
So, summary:
 - updated appropriate *.nix files, checked that it work well, tested with nixos 22.11 and 22.05
 - device posture feature that works - firewall, disk encryption, **fetching last OS upgrade date doesn't implemented**
 - autostart of client doesn't work and didn't work before(will pay attention on this, but as @tonyshkurenko said to me it was blocker from nixos side)
 - we can't merge this PR to master branch because  current [twingate/default.nix](https://github.com/Twingate/nixpkgs/pull/4/files#diff-c5ace1e9ea3e3a7e434729930ede28c9f10ece7b8af2010332316602a8a1c781R19-R20) contains link that refers to build of [rc/nixos-support](https://github.com/Twingate/client-linux/tree/rc/nixos-support). So, we should release new version of twingate client, after that update link and sha to master

cc: @tonyshkurenko

**Comment by alexprivalov** [comment]
New update. Quick approach to install twingate package without involving long `nixos-rebuilt switch`:
 - `NIXPKGS_ALLOW_UNFREE=1 nix-build -E 'with import <nixpkgs> {}; callPackage /media/sf_alex/dev/twingate/nixpkgs/pkgs/applications/networking/twingate/default.nix {}'`
 - `sudo nix-env -i /nix/store/f47b3w6nrz8m176bxy8dga8js9xs4i0v-twingate-1.0.58	#last line of output nix-build`
image

File: pkgs/applications/networking/twingate/default.nix
Comment (sagie-shamay) [inline pkgs/applications/networking/twingate/default.nix:16]: The latest version is `1.0.60`

File: maintainers/maintainer-list.nix
Comment (sagie-shamay) [inline maintainers/maintainer-list.nix:13067]: Should we add @alexprivalov as a maintainer?

**Review by sagie-shamay — APPROVED** [review]

**Pull request merged**
